common/src/types.rs

```diff
@@ -97,13 +98,17 @@ pub enum Diff {
     General(Box<ReservedState>, Hash256),
 }
 
+/// A general transaction to be included in the agenda.
+///
+/// Note that `author` and `timestamp` of a general transaction is not cared by the Simperby protocol;
+/// user may set it to whatever they want.
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
 pub struct Transaction {
-    pub author: PublicKey,
-    pub timestamp: Timestamp,
     pub head: String,
     pub body: String,
     pub diff: Diff,
+    pub author: MemberName,
+    pub timestamp: Timestamp,
```

Comment:
Any reason for relocating these fields? 

It might be trivial but I think it's better to keep it the same way since the orders used in below tests.

Reply:
reverted